Add 4 Digit osu!mania World Cup 4 article (#6751)
ppy/osu-wiki: wiki/Tournaments/4DM/4/en.md

| Designer | ![][flag_NL] [DannyPX](https://osu.ppy.sh/users/11253722), ![][flag_SG] [Polytetral](https://osu.ppy.sh/users/8612061) |
| Statistician | ![][flag_NL] [Albionthegreat](https://osu.ppy.sh/users/9853595), ![][flag_NL] [DannyPX](https://osu.ppy.sh/users/11253722) |
| Wiki editor | ![][flag_NL] [Albionthegreat](https://osu.ppy.sh/users/9853595), ![][flag_US] [chxu](https://osu.ppy.sh/users/13712190) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1452835)
- [4DM Discord server](https://discord.gg/NzYtsxV)
- [4DM YouTube channel](https://youtube.com/channel/UC9mLxo8frxAJCf1vrASEFDg)
- [Livestream](https://twitch.tv/4digitmwc)
- [Tournament bracket](https://challonge.com/4DMWC4)
- [Pick'em predictions website](https://pickem.hwc.hr/tournaments/77) hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)

> - Spreadsheet

Reviewer: only tournament with this formatting but ok

```suggestion
- Spreadsheets
```

| #11 | ![][flag_SG] Singapore | 12.0 |
| #12 | ![][flag_PE] Peru | 14.0 |
| #13 | ![][flag_GB] United Kingdom | 14.6 |
| #14 | ![][flag_SE] Sweden | 14.8 |
| #15 | ![][flag_DE] Germany | 15.0 |
| #16 | ![][flag_ES] Spain | 15.6 |
| #17 | ![][flag_TW] Taiwan | 15.8 |
| #18 | ![][flag_AU] Australia | 16.2 |
| #19 | ![][flag_JP] Japan | 17.0 |
| #20 | ![][flag_MX] Mexico | 17.0 |
| #21 | ![][flag_RU] Russian Federation | 19.8 |
| #22 | ![][flag_PL] Poland | 21.2 |
| #23 | ![][flag_HK] Hong Kong | 21.8 |
| #24 | ![][flag_VE] Venezuela | 23.4 |
| #25 | ![][flag_BE] Belgium | 23.4 |
| #26 | ![][flag_CL] Chile | 25.2 |
| #27 | ![][flag_DK] Denmark | 25.4 |
| #28 | ![][flag_AR] Argentina | 25.4 |
| #29 | ![][flag_IT] Italy | 25.6 |
| #30 | ![][flag_DO] Dominican Republic | 26.6 |
| #31 | ![][flag_FI] Finland | 31.2 |
| #32 | ![][flag_PA] Panama | 31.6 |
| #33 | ![][flag_EC] Ecuador | 31.6 |
| #34 | ![][flag_UA] Ukraine | 31.8 |
| #35 | ![][flag_GT] Guatemala | 34.6 |
| #36 | ![][flag_AE] United Arab Emirates | 35.4 |
| #37 | ![][flag_NI] Nicaragua | 35.4 |
| #38 | ![][flag_UY] Uruguay | 35.6 |
| #39 | ![][flag_ZA] South Africa | 38.8 |
| #40 | ![][flag_CO] Colombia | 40.0 |

> ## Structure

Reviewer: ```suggestion
## Ruleset
```

Teams will be ranked based on the “score rating” of their qualifier performances, meaning that each teams' performance on each of the five qualifier maps will be ranked compared to every other team and the ranks that each team achieves on these five maps will then be added up. The teams will then be sorted based on their "score rating" from lowest to highest, with lower score ratings being ranked better than higher score ratings. Essentially, a team's qualifier performance is ranked based on how well they perform compared to every other participating team.

The results of the Qualifier Stage will also be used to determine the “seed” of each team.

### Match procedures

After Qualifiers are over and every team has either gotten a seed or not qualified, the bracket will then begin. 4DM4 will use a standard Double Elimination bracket where the top seeded team will face the bottom seeded team, second highest will face second lowest, so on and so forth. Each team must lose two different matches to be eliminated from the tournament, this means that every team will get the chance to at least play two separate pools.

**Matches will be played with the following rules:**

> - Team captains will be notified of their match via ping on discord by the referee of their match 30 minutes before the match is scheduled to commence. At this time, team captains shall give the link of one map they wish to play as warmup to the referee of their match. This map must be an osu!mania 4k map and must be no longer than 5 minutes. Referees may refuse a teams warmup and demand another one from the team captain if the map does not meet these guidelines or is found to be offensive. Failure to provide a warmup to the referee at this time may result in this team skipping their warmup map.

Reviewer: ```suggestion
- Team captains will be notified of their match via ping on Discord by the referee of their match 30 minutes before the match is scheduled to commence. At this time, team captains shall give the link to one map they wish to play as warmup to the referee of their match. This map must be an osu!mania 4k map and must be no longer than 5 minutes. Referees may refuse a team's warmup map and demand another one from the team captain if it does not meet these guidelines or is found to be offensive. Failure to provide a warmup to the referee may result in this team skipping their warmup map.
```

- Team captains will be invited to their match lobby by a referee no less than 5 minutes before the match is scheduled to commence. In the time before the match, team captains will invite and gather their teams into the lobby.

> - When both team members have 3 members present in the lobby, the match will commence by each team playing their selected warmup map.

Reviewer: ```suggestion
- When both teams have 3 members present in the lobby, the match will commence starting with warmups.
```

> - After warmups, both team captains will !roll in the lobby chat. The captain with the lower of the two rolls will choose a map in the freemod pool to ban from play, then the captain with the higher roll will choose a map in the freemod pool to ban from play, as well as their first pick to play from the freemod pool.

Reviewer: ```suggestion
- After warmups, both team captains will `!roll` in the lobby chat. The captain who rolled lower will choose a map to ban from the FreeMod pool, and then the other captain will do the same, as well as be the first to pick a map to play from the FreeMod pool.
```

- Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready.

> - Each team will receive one "tactical timeout" of two minutes, to be used as extra time to pick or ban a beatmap. The tactical timeout may only be used by the team who has to pick and can also be used when getting ready.

Reviewer: ```suggestion
- Each team will receive one tactical timeout of two minutes, to be used as extra time to pick or ban a beatmap. The tactical timeout may only be used by the team who has to pick and can also be used when getting ready.
```

> - If a tiebreaker gets played, both teams may use their tactical timeout, but only one timeout can be given.

Reviewer: ```suggestion
- If a tiebreaker map gets played, both teams may use their tactical timeout, but only one timeout can be given.
```

- After each map concludes, the team with the highest team score on the map will be awarded one point. Picks will alternate between team captains until a winner is decided by being the first to get the required amount of points to win that round.
- If team scores on a map are tied, no points will be awarded and the teams will play the map again.
- In the Round of 32 and Round of 16, teams need to win 5 maps to win a match (best of 9). In the Quarterfinals and Semifinals, teams need to win 6 maps to win a match (best of 11). In the Finals and Grand Finals, teams need to win 7 maps to win the match (best of 13).

> - If both teams tie at match point, the tiebreaker map will be played, the team who wins the tiebreaker map will win the match.

Reviewer: ```suggestion
- If both teams tie at match point, the tiebreaker map will be played, and the team who wins the tiebreaker map will win the match.
```

### Additional Rules

> The tournament hosts are responsible for scheduling all matches and informing players of when they are scheduled to play. If a team is unhappy with when they are scheduled by the hosts to play, they have the option to reschedule their match as late as the end of Thursday of the week before the weekend of their match. Instructions on how to do so will be provided in the discord server.

Reviewer: Thursday in what timezone?

```suggestion
The tournament hosts are responsible for scheduling all matches and informing players of when they are scheduled to play. If a team is unhappy with their scheduled time, they have the option to reschedule their match as late as the end of Thursday of the week before the weekend of their match. Instructions on how to do so will be provided in the Discord server.
```

> Please note that while the organizers will try to schedule all matches for the best possible time for both teams, timezone conflicts may make some matches be forced to occur in the early mornings or late nights.

Reviewer: ```suggestion
Please note that while the organisers will try to schedule all matches for the best possible time for both teams, timezone conflicts may force some matches to occur in the early mornings or late nights.
```

- If no referee is available to referee a match, the match shall be delayed.
- If no streamers are available to broadcast the match, the match will still happen without being broadcasted.
- If no commentators are available to commentate the match but broadcasters are available to broadcast the match, the match will still be broadcasted but without commentary.

> Please note that while 4DM4 will try to broadcast every match, certain matches may be unable to be broadcasted due to overlapping matches and/or lack of streamer availability. Rescheduled matches will be prioritized under non-rescheduled matches in terms of being broadcasted.

Reviewer: ```suggestion
Please note that certain matches may be unable to be broadcasted due to overlapping matches and/or lack of streamer availability. Priority will be given to non-rescheduled matches.
```

> Each respective stage of the tournament is to be completed in a single weekend, using the same map pool for the entire weekend. Weekday matches are allowed if both team captains agree to them and they take place before the weekend of that scheduled stage. As aforementioned, map pools will be released roughly a week before the stage they are to be played on. The latest a map in a released map pool may be slightly updated (for example, as to fixing a notable timing or snapping issue) is by 23:59UTC of the Monday after the map pool is released.

Reviewer: ```suggestion
Each respective stage of the tournament is to be completed in a single weekend, using the same mappool for the entire weekend. Weekday matches are allowed if both team captains agree to them and they take place before the weekend of that scheduled stage. As mentioned before, mappools will be released roughly a week before the stage they are to be played on. The latest a map in a released mappool may be slightly updated (for example, to fixing a notable timing or snapping issue) by 23:59 UTC of the Monday after the mappool is released.
```

> Tournament hosts have the right to revoke any signed player’s eligibility to participate in the tournament due to an obvious excess of skill compared to the tournament’s target demographic, or due to blacklist-worthy community offenses that may not see them screened out of the tournament in the Screening Phase.

Reviewer: Note that the rules for [official tournament support](https://osu.ppy.sh/wiki/en/Tournaments/Official_support#tournaments) (this tournament is badged as it seems) require "ample evidence" and approval from the account support team in order to reject registrants that pass screening, though I suppose the next paragraph covers this

```suggestion
Tournament hosts have the right to revoke any signed player's eligibility to participate in the tournament due to an obvious excess of skill compared to the tournament's target demographic, or due to blacklist-worthy community offenses that may not see them screened out of the tournament in the screening phase.
```

> Please note that tournament hosts will only enact restrictions of this caliber in the most extreme of scenarios for reasons seen as precautionary protection of the tournament’s fairness to its players, as the hosts acknowledge that the reasons for doing this are inherently very subjective and broad.

Reviewer: ```suggestion

Please note that tournament hosts will only enact restrictions of this calibre in the most extreme of scenarios for reasons seen as precautionary protection of the tournament's fairness to its players, as the hosts acknowledge that the reasons for doing this are inherently very subjective and broad.
```

> Repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chatroom, the official discord server, or the official twitch channel chatroom is strictly prohibited and will lead to, in successive order with host’s discretion to skip certain steps:

Reviewer: ```suggestion
Repeated and excessive toxic behavior towards any staff member, player, or spectator either in match multiplayer lobby, the official Discord server, or the official Twitch channel is strictly prohibited and will lead to, in successive order or with a few steps skipped at the host's discretion:
```

1. Casual, undocumented warning
2. Formal, documented warning

> 3. Timeout of player wherever necessary for a subjective amount of time (Players on timeout during tournament match will be disallowed to play during their match and must be swapped out for other players)

Reviewer: ```suggestion
3. Timeout of a player wherever necessary for a subjective amount of time. Players on timeout during tournament match will be disallowed to play during their match and must be swapped out for other players.
```

> 4. Official ban from the tournament as well as a ban from the official discord server and twitch channel

Reviewer: ```suggestion
4. Official ban from the tournament as well as a ban from the official Discord server and Twitch chat.
```

Please note that tournament hosts will only enact restrictions of this calibre in the most extreme of scenarios for reasons seen as precautionary protection of the tournament's fairness to its players, as the hosts acknowledge that the reasons for doing this are inherently very subjective and broad.

Repeated and excessive toxic behavior towards any staff member, player, or spectator either in match multiplayer lobby, the official Discord server, or the official Twitch channel is strictly prohibited and will lead to, in successive order or with a few steps skipped at the host's discretion:

1. Casual, undocumented warning
2. Formal, documented warning
3. Timeout of a player wherever necessary for a subjective amount of time. Players on timeout during tournament match will be disallowed to play during their match and must be swapped out for other players.
4. Official ban from the tournament as well as a ban from the official Discord server and Twitch chat.

> If one player on a team has their account restricted while still participating in the tournament for reasons of believed cheating, multiaccounting, or anything else, this player's entire team shall be disqualified from the tournament with no exceptions. Depending on the case, the team most recently defeated in a match by the newly disqualified team may be allowed to advance in this disqualified team's place. If this happens, the result of this former match will be voided.

Reviewer: ```suggestion
If one player on a team has their account restricted while participating in the tournament, for reasons of believed cheating, multiaccounting, or anything else, their entire team shall be disqualified from the tournament with no exceptions. Depending on the case, the team most recently defeated in a match by the newly disqualified team may be allowed to advance in this disqualified team's place. If this happens, the result of this former match will be voided.
```
